Remotion/Web/Core/ExecutionEngine/WxeReturnOptions.cs

```diff
@@ -29,7 +30,7 @@ namespace Remotion.Web.ExecutionEngine
   [Serializable]
   public sealed class WxeReturnOptions : INullObject
   {
-    public static readonly WxeReturnOptions Null = new WxeReturnOptions (false, null);
+    public static readonly WxeReturnOptions Null = new WxeReturnOptions (false, null!);
```

Comment:
we can (& should) make CallerUrlParameters notnull if IsReturning==true

Reply:
It's currently not possible to implement this kind of relation between parameters.